--- xplc-0.3.13/config/rules.mk	2005-10-31 14:33:26.000000000 -0800
+++ xplc-0.3.13/config/rules.mk.new	2007-04-26 12:55:53.000000000 -0700
@@ -107,19 +107,18 @@ installdirs:
 	mkdir -p $(DESTDIR)$(bindir)
 	mkdir -p $(DESTDIR)$(mandir)/man1
 	mkdir -p $(DESTDIR)$(libdir)/pkgconfig
-	mkdir -p $(DESTDIR)$(libdir)$(xplcdir_version)
 	mkdir -p $(DESTDIR)$(includedir)$(xplcdir_version)/xplc
 
 install: default installdirs
 	$(INSTALL_PROGRAM) libxplc.so $(DESTDIR)$(libdir)/libxplc.so.$(PACKAGE_VERSION)
-	$(INSTALL_DATA) libxplc.a $(DESTDIR)$(libdir)$(xplcdir_version)
-	$(INSTALL_DATA) libxplc-cxx.a $(DESTDIR)$(libdir)$(xplcdir_version)
+	$(INSTALL_DATA) libxplc.a $(DESTDIR)$(libdir)
+	$(INSTALL_DATA) libxplc-cxx.a $(DESTDIR)$(libdir)
 	$(INSTALL_DATA) dist/xplc.pc $(DESTDIR)$(libdir)/pkgconfig/xplc$(pc_version).pc
 	$(INSTALL_DATA) $(wildcard include/xplc/*.h) $(DESTDIR)$(includedir)$(xplcdir_version)/xplc
 	$(INSTALL_PROGRAM) uuid/bin/uuidgen $(DESTDIR)$(bindir)
 	$(INSTALL_DATA) uuid/bin/uuidgen.1 $(DESTDIR)$(mandir)/man1
-	$(LN_S) $(lib_prefix_version)libxplc.so.$(PACKAGE_VERSION) $(DESTDIR)$(libdir)$(xplcdir_version)/libxplc.so
-	$(LN_S) libxplc.a $(DESTDIR)$(libdir)$(xplcdir_version)/libxplc_s.a
+	$(LN_S) libxplc.so.$(PACKAGE_VERSION) $(DESTDIR)$(libdir)/libxplc.so
+	$(LN_S) libxplc.a $(DESTDIR)$(libdir)/libxplc_s.a
 	$(LN_S) uuidgen $(DESTDIR)$(bindir)/uuidcdef
 	$(LN_S) uuidgen.1 $(DESTDIR)$(mandir)/man1/uuidcdef.1
 
